.github/workflows/release.yaml

```diff
@@ -25,6 +25,8 @@ jobs:
       uses: actions/setup-go@d35c59abb061a4a6fb18e82ac0862c26744d6ab5 # tag=v5.5.0
       with:
         go-version: ${{ steps.vars.outputs.go_version }}
+        check-latest: true
```

Comment:
My understanding is that this will actually slow down the CI by looking for newer go versions, rather than pinning to the version we have defined. I'm not sure we would want to be constantly pulling newer go versions anyway, typically we pin to a known working version and then update periodically from there

Reply:
Agree